app.css

```diff
@@ -267,15 +272,133 @@ footer {
   align-items: center;
   text-align: center;
 }
-.contact-button:hover {
+.cont-button:hover {
   background-color: #4c73d6;
   color: white;
 }
-.contact-button:active {
+.cont-button:active {
   background-color: #4c73d6;
   color: white;
 }
-.contact-button:disabled {
+.cont-button:disabled {
   background-color: #6276a8;
   color: white;
 }
+
+@media screen and (max-width: 720px) {
+  .nav-link {
+    display: none;
+  }
+  .logo-info {
+    display: block;
+  }
+}
+
+@media screen and (min-width: 720px) {
+  .personal-info-detail p {
+    margin: 10px 100px;
+  }
```

Comment:
- [ ] In the Intro section of your portfolio, kindly ensure you have the appropriate background image as we have on the template.

- [ ] Kindly make your **`'I'm Freddy Glad to see to you'`** text bigger, as we have on the template

- [ ] The **'Glad to see you!'** text should be on the next line as we have on the template, kindly do that.

- [ ] As I have pointed out earlier, kindly ensure you remove the unnecessary space on the right side of you website.
  Your Implementation | Figma Design
  :---:|:---:
  ![freddy-intro](https://user-images.githubusercontent.com/36542986/161252506-118f59e6-f11a-4faf-9e62-3bfef6d43ea7.jpg) | ![freddy-intro-figma](https://user-images.githubusercontent.com/36542986/161252511-563f3f6c-4be2-4552-a776-a60d83151374.jpg)